scopes/html/html-to-react.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useRef, useEffect } from 'react';
+import { RenderHtmlComposition } from './utils';
+
+/** adapter that converts html elements to react elements */
+
+export function htmlToReact(element: HTMLElement) {
+	return HtmlToReactAdapter;
```

Comment:
better to write it as annonymous in this case
so 
```js
return () => {
		const ref = useRef<HTMLDivElement>(null);
		useEffect(() => { RenderHtmlComposition(ref.current, element) }, []);
		return <div ref={ref} />
```

Reply:
it would look better in React dev tools, no?